### PR TITLE
Russell-L232.prc_nonco2

### PR DESCRIPTION
**Pull request by russellhz**
#509

### CODE REVIEW

File: R/zchunk_L232.prc_nonco2.R [hunk 1]
```diff
@@ -27,72 +27,97 @@ module_emissions_L232.prc_nonco2_DISABLED <- function(command, ...) {
...
     all_data <- list(...)[[1]]
 
     # Load required inputs
     GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
-    A_regions <- get_data(all_data, "emissions/A_regions")
-    L131.nonco2_tg_R_prc_S_S_Yh <- get_data(all_data, "L131.nonco2_tg_R_prc_S_S_Yh")
+    A_regions <- get_data(all_data, "emissions/A_regions") %>%
+      select(region, SO2_name)
+    L131.nonco2_tg_R_prc_S_S_Yh <- get_data(all_data, "L131.nonco2_tg_R_prc_S_S_Yh") %>%
+      mutate(year = as.integer(year))
```
Comment (bpbond) [inline R/zchunk_L232.prc_nonco2.R]: Let's fix this upstream (in L131).

File: inst/extdata/emissions/A32.max_reduction.csv [hunk 2]
```diff
@@ -0,0 +1 @@
...
urban processes,wastewater treatment,wastewater treatment,,,,,,
urban processes,aerosols,aerosols,,,,,,
urban processes,fire_exting,fire_exting,,,,,,
industrial processes,nitric acid,nitric acid,,,,,,
industrial processes,foams,foams,,,,,,
industrial processes,solvents,solvents,,,80,,,60
industrial processes,HCFC_22_Prod,HCFC_22_Prod,,,,,,
industrial processes,semiconductors,semiconductors,,,,,,
industrial processes,Al_Mg,Al_Mg,95,95,80,,,60
industrial processes,other industrial processes,other industrial processes,99,95,80,,,60
```
Comment (cahartin) [inline inst/extdata/emissions/A32.max_reduction.csv]: add 'r' to base-year.
Do we know what the base year is?

File: inst/extdata/emissions/A32.steepness.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
urban processes,wastewater treatment,wastewater treatment,,,,,,
urban processes,aerosols,aerosols,,,,,,
urban processes,fire_exting,fire_exting,,,,,,
industrial processes,nitric acid,nitric acid,,,,,,
industrial processes,foams,foams,,,,,,
industrial processes,solvents,solvents,,,3.5,,,3.5
industrial processes,HCFC_22_Prod,HCFC_22_Prod,,,,,,
industrial processes,semiconductors,semiconductors,,,,,,
industrial processes,Al_Mg,Al_Mg,3.5,3.5,3.5,,,3.5
industrial processes,other industrial processes,other industrial processes,3.5,3.5,3.5,,,3.5
```
Comment (cahartin) [inline inst/extdata/emissions/A32.steepness.csv]: can you add what journal the Smith et al., 2005 is from?

**Comment by cahartin** [comment]
@d3y419 do you mind reviewing? The emissions group could use an expert to review the remaining chunks for their science.

**Comment by bpbond** [comment]
The one failing check above was due to a timeout. Merging.

**Pull request merged**